cdn/dev/js/kmwlive.js

```diff
@@ -272,7 +272,14 @@ function refreshButtons() {
   var len=getKMWInputLength('message'); // returns -1 for desktop page element
 
   if(len < 0) {
-    len=$('#message').val().length;
+    // refreshButtons runs via setInterval.  It is possible for one call to persist through page reload,
+    // in which case the #message element is unavailable.
+    var message = $('#message').val();
+    if(!message) {
+      return;
```

Comment:
Does this prevent buttons from getting reset? ll 286-292

Reply:
It does, though I believe this in a specific case where it would be _impossible_ to reset them.  If the `#message` element is inaccessible, the others likely are as well.